src/components/diagnosisForm/QuestionItem.js

```diff
@@ -89,5 +90,6 @@ DiagnosisQuestionItem.propTypes = {
     content: PropTypes.string.isRequired,
   }).isRequired,
   index: PropTypes.number.isRequired,
+  defaultAnswer: PropTypes.oneOf([-1, 0, 1]).isRequired,
```

Comment:
Warning: Failed prop type: The prop `defaultAnswer` is marked as required in `<<anonymous>>`, but its value is `undefined`.